engine/src/main/java/io/camunda/zeebe/engine/processing/bpmn/event/StartEventProcessor.java

```diff
@@ -75,4 +70,13 @@ public void onTerminate(final ExecutableStartEvent element, final BpmnElementCon
     incidentBehavior.resolveIncidents(terminated);
     stateTransitionBehavior.onElementTerminated(element, terminated);
   }
+
+  private BpmnElementContextImpl buildContextForFlowScopeInstance(
+      final BpmnElementContext context) {
+    final var flowScopeInstance = stateBehavior.getFlowScopeInstance(context);
+    final var flowScopeInstanceContext = new BpmnElementContextImpl();
+    flowScopeInstanceContext.init(
+        flowScopeInstance.getKey(), flowScopeInstance.getValue(), flowScopeInstance.getState());
+    return flowScopeInstanceContext;
+  }
```

Comment:
Couple of reasons: 
* Keeping the change as small as possible for backporting
* There already is an `init(...)` method which screams like it should just take an element instance. So probably somebody had thought about it and decided against it
* Overall, I don't understand the pattern of empty constructor and init method and I feel that adding a convenience constructor would jeopardize the intended design. In particular, it seems deliberate to me that this class has no dependency to `ElementInstance` 
* In case of doubt, I aim to use what is there instead of making changes to it